crates/orchestrator/src/lib.rs

```diff
@@ -317,7 +323,7 @@ where
             return Ok((*node_index, *is_da));
         }
 
-        if !self.accepting_new_keys {
+        if !self.config.disable_registration_verification && !self.accepting_new_keys {
```

Comment:
Was the `disable_registration_verification` check meant to go in the other conditional? (I don't think it should be here)